lessons/topics/01-variables/README.md

```diff
@@ -0,0 +1,20 @@
+# Variables, Scope, Hoisting
+
+In this chapter we will talk about how variables behave in different contexts, how the scope affect the reach of said chunks of data and how we can manage to 
+move the data between blocks and structures
+
+## Previous Knowledge
+
+List or links to this repository for previous Knowledge topics eg.:
+
+* variables Assignment
+* What is a constant? And how do they work?
+* Scope
+* Hoisting
+* Simple Functions
+
+## References
+
+* [Hoisting Javascript](https://developer.mozilla.org/en-US/docs/Glossary/Hoisting)
+
```

Comment:
Maybe you could add some references to scope and variables

Reply:
Here are other links that could be good to add:
- [Variables](https://javascript.info/variables)
- [The old var](https://javascript.info/var)
- [Variable scope, closure](https://javascript.info/closure)
- [Global object] (https://javascript.info/global-object)


It would be good too if we talk/write about the closure concept, it means a lot on the hoisting and scope topic

Reply:
i added some new links